web/src/routes/(user)/people/[personId]/+page.svelte

```diff
@@ -58,12 +59,24 @@
   let people = data.people.people;
   let personMerge1: PersonResponseDto;
   let personMerge2: PersonResponseDto;
+  let potentialMergePeople: PersonResponseDto[] = [];
 
   let personName = '';
 
+  let name: string;
+  let suggestedPeople: PersonResponseDto[] = [];
+
   $: isAllArchive = Array.from($selectedAssets).every((asset) => asset.isArchived);
   $: isAllFavorite = Array.from($selectedAssets).every((asset) => asset.isFavorite);
 
+  $: {
+    suggestedPeople = !name
+      ? []
+      : people
+          .filter((person: PersonResponseDto) => person.name.startsWith(name) && person.id !== data.person.id)
```

Comment:
Let's convert the input name and names' candidate to lowercase for better suggestion support in the filter method.